source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ACRCustomImageResolver.h

```diff
@@ -0,0 +1,15 @@
+//
+//  ACRCustomImageResolver.h
+//  ADCIOSVisualizer
+//
+//  Created by Harika P on 10/11/25.
+//  Copyright © 2025 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <AdaptiveCards/AdaptiveCards.h>
```

Comment:
Do we need this imports?
Please examine elsewhere. Imports are low hanging fruits to improve compiler performance.